.github/workflows/.reusable-docker-e2e-tests.yml

```diff
@@ -27,6 +27,10 @@ on:
         description: The runner label to use. Defaults to `ubuntu-latest`
         required: false
         default: ubuntu-latest
+    secrets:
+      gcr-token:
+        description: A token to use for logging into Github Container Registry. If not provided, login does not occur.
+        required: false
```

Comment:
Shouldn't required be set to true?

Reply:
No. GCR login is not required (and, in fact, is not possible due to blocked secrets) for external PRs.